scheds/rust/scx_layered/src/bpf/main.bpf.c

```diff
@@ -2658,7 +2678,7 @@ static void refresh_cpus_flags(struct task_ctx *taskc,
 
 		if (!(nodec = lookup_node_ctx(node_id)) ||
 		    !(node_cpumask = cast_mask(nodec->cpumask)))
-			return;
+			break;
```

Comment:
This is `scx_bpf_error()` condition. There's no point in continuing.

Reply:
made this an explicit error instead of just a return, thanks.